### ISSUE TITLE
USER STORY: Admin add on items and control

### ISSUE THREAD

**Issue by Danger0101** [Danger0101/Blossom_Haven_CI_PP5#13]
As an **Admin**, I want to **manage the availability and pricing of add-on items** so that they can only be added if they are in stock.

AC1 - Admin dashboard includes options to add, edit, or delete add-on items.
AC2 - Add-on management page displays a list of add-ons with options to adjust their availability and prices.
AC3 - If the addon is out of stock I want users to be notified visually with an out-of-stock message.
AC4 - Add-on items can be set in edit or add products on main site.

**Comment by Danger0101** [comment]
Added AC3 to the list for more overall coverage as this is nearly complete it will be added in with the mobile version of my addon item display. I managed to solve the adding to cart to fit my needs without over complicating the entire system. I am happy with the result. Next up is mobile and desktop variation to UI visuals of the addon products to suit each one appropriately. Overall, my test version is okay for desktops but will need some major tweaks to be satisfying although it was only a temporary solution for testing purposes, I think it might be mostly kept as is but tweaked. 

For mobiles I am looking at a smaller card like version with possibly arrows to cycle through the products similar to my Burger blast project. As I think it will give users a good idea of the items. Will confirm once I find a way I think fits best. I scrapped my old idea as practicality of it wasn't a good fit in the end.
